### ISSUE TITLE
Raise visibility of the publishing journal

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#2292]
**Raise visibility of the publishing journal**

The European Journal of Taxonomy is publishing taxonomic articles in a classic pdf workflow and finally pass it on to Plazi to do the taxonomic markup and registration with GBIF. Here is an example in GBIF: 

`https://www.gbif.org/dataset/d72b5d68-b59f-42c6-963f-13f3629fa332`

The journal unfortunately is hardly visible on that page and Plazi way more prominent and attributed. For Pensoft journals  a different route is followed such that each journal is a publisher on GBIF:
`https://www.gbif.org/publisher/750a8724-fa66-4c27-b645-bd58ac5ee010`

That obviously renders the journals way more nicely for each article, but requires the journals themselves to do the markup and registration with GBIF:
`https://www.gbif.org/dataset/cdd9d4c9-6eb1-4616-8693-727e4fb1f407`

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/213d29f8504ffc819b040709b0981483:ba8b58b3ad601b9f9641937317deeebfa1bd74a7e206b87d9b32dc09d3828a097b28b8f84a303240f2a62c556d8f807a1929c26a50b6a5145c77337765c65fa1)
System: Chrome 78.0.3904 / Mac OS X 10.15.1
Referer: https://www.gbif.org/dataset/d72b5d68-b59f-42c6-963f-13f3629fa332
Window size: width 1649 - height 906
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-11-15T07:50:08.883Z',mode:absolute,to:'2019-11-15T07:56:08.883Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2019-11-15T07:50:08.883Z',mode:absolute,to:'2019-11-15T07:56:08.883Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
There is nothing in the data that really allows us to do it differently on the website (not that I can see/think of). And we use the fact that it is published by Plazi quite a lot to do custom 'hacks' like calling it a treatment etc. Should we have another type of datasets/metadata and corresponding index for these kind of datasets?

**Comment by cjchester** [comment]
Indeed, apart from the DOI and citation, EJT isn't identified anwhere as a publisher or data provider, meaning that we can't take advantage of tools such as the literature tracking.

It's important that these data sets are identified as provided by Plazi as they are indeed an interpretation of the original publications, and we do find the "description" fair and transparent, e.g.

This dataset contains the digitized treatments in Plazi based on the original journal article Meisch, Claude, Smith, Robin J., Martens, Koen (2019): A subjective global checklist of the extant non-marine Ostracoda (Crustacea). European Journal of Taxonomy 492: 1-135, DOI: https://doi.org/10.5852/ejt.2019.492

Is there some way we could be identified as a publisher while retaining the link with Plazi?

**Comment by cjchester** [comment]
EJT now includes links from its website to the equivalent datasets on GBIF (via a DOI search request)! 
It would be really useful to have a "publisher" status and to access the metrics of the citation tracker - this would be a great thing to show our partners and funders. Is there any way we can do that?
